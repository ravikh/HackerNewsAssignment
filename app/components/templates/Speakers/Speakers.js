import React from 'react';
import PropTypes from 'prop-types';

const Speakers = () => {
  return <>Speakers</>;
};

Speakers.propTypes = {
  speakersData: PropTypes.array,
};

export default Speakers;
