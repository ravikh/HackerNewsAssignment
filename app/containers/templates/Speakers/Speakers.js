import React from 'react';

import SpeakersComponent from '../../../components/templates/Speakers';

const Speakers = props => <SpeakersComponent {...props} />;

export default Speakers;
