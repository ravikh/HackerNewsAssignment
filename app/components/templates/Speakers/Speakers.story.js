import React from 'react';
import { storiesOf } from '@storybook/react';
import Speakers from './index';

storiesOf('Speakers', module).add('Speakers', () => <Speakers />);
