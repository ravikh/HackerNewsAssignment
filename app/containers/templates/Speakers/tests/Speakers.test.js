import React from 'react';
import { shallow } from 'enzyme';

import Speakers from '../index';

const speakersData = [
  {
    name: 'test name',
    title: 'test title',
  },
];

describe('Speakers component', () => {
  test('should render correctly', () => {
    const SpeakersComponent = shallow(<Speakers speakersData={speakersData} />);
    expect(SpeakersComponent).toMatchSnapshot();
  });
});
