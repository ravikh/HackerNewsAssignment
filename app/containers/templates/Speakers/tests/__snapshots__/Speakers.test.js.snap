// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Speakers component should render correctly 1`] = `
<Speakers
  speakersData={
    Array [
      Object {
        "name": "test name",
        "title": "test title",
      },
    ]
  }
/>
`;
